prow/tide/tide.go

```diff
@@ -801,25 +802,15 @@ func (c *Controller) pickBatch(sp subpool, cc contextChecker) ([]PullRequest, er
 }
 
 func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
-	var merged []int
+	var merged, failed []int
 	defer func() {
 		if len(merged) == 0 {
 			return
 		}
 		tideMetrics.merges.WithLabelValues(sp.org, sp.repo, sp.branch).Observe(float64(len(merged)))
 	}()
 
-	augmentError := func(e error, pr PullRequest) error {
-		var batch string
-		if len(prs) > 1 {
-			batch = fmt.Sprintf(" from batch %v", prNumbers(prs))
-			if len(merged) > 0 {
-				batch = fmt.Sprintf("%s, partial merge %v", batch, merged)
-			}
-		}
-		return fmt.Errorf("failed merging #%d%s: %v", int(pr.Number), batch, e)
-	}
-
+	var errs []error
 	log := sp.log.WithField("merge-targets", prNumbers(prs))
 	maxRetries := 3
```

Comment:
`const maxRetries = 3` (and can you move this down to the for loop where it is used)?

Reply:
Done.